### DISCUSSION TITLE
Table mode move row up/down

### DISCUSSION THREAD

**Discussion by secrn** [josdejong/svelte-jsoneditor#365] [Ideas]
Hello, table mode works great. Is it possible to move an existing row up/down. Thanks

**Comment by josdejong** [comment]
Thanks for your suggestion. Editing in table mode is still relatively limited, I want to add more options to edit rows and columns. Dragging a row up/down (or select multiple rows and drag them all up/down) would be great indeed 😎

**Comment by secrn** [comment]
Thank you for considering it.

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it. Help implementing this would be welcome.
